### DISCUSSION TITLE
Status of SQLITE_ENABLE_JSON1

### DISCUSSION THREAD

**Discussion by ShaharHD** [fnc12/sqlite_orm#1314] [Q&A]
What is the maturity and/or stability of `SQLITE_ENABLE_JSON1` ?

It is not documented, but I can see it in the `sqlite_orm.h` file.

**Comment by fnc12** [comment]
it is enabled always in SQLite from specific version (I don't remember exactly). `sqlite_orm` has its support if `SQLITE_ENABLE_JSON1` is defined.
